wikipron/config.py

```diff
@@ -81,6 +82,7 @@ def __init__(
         )
         self.skip_spaces_word: bool = skip_spaces_word
         self.skip_spaces_pron: bool = skip_spaces_pron
+        self.parens: bool = parens
```

Comment:
What do you think about `self.keep_parens: bool = keep_parens` instead of `self.parens: bool = parens`? Kyle's comment about other potential features related to parens makes me think we probably should be proactive about using more explicit names around earlier.

Reply:
This is reasonable. I just thought `parens` fit in with the existing config attributes, but I can make the change to make it more clear—probably more important.

Reply:
In fact maybe `skip_parens` sould be better yet?

Reply:
Good point. I'll reverse the logic and rename the flags `--skip-parens` and `--no-skip-parens` to keep it consistent with the other flags.